staging/src/k8s.io/api/networking/v1/types.go

```diff
@@ -52,31 +52,38 @@ const (
 
 // NetworkPolicySpec provides the specification of a NetworkPolicy
 type NetworkPolicySpec struct {
-	// Selects the pods to which this NetworkPolicy object applies. The array of
-	// ingress rules is applied to any pods selected by this field. Multiple network
-	// policies can select the same set of pods. In this case, the ingress rules for
-	// each are combined additively. This field is NOT optional and follows standard
+	// Selects the pods to which this NetworkPolicy object applies for ingress and/or egress,
```

Comment:
api fields should start with the field name as seen by the user.

```
podSelector selects the pods ...
```

This is often inconsistently applied, but in general if you change the field docs and touch the first sentence I always ask for it to be fixed.  So for this PR podSelector, ingress, and egress should all have their first sentence corrected.